windows.fnl

```diff
@@ -3,9 +3,13 @@
         : count
         : concat
         : contains?
+        : apply
```

Comment:
Seems we don't need apply, map, logf, or contains? anymore.

Reply:
I'm planning to bring an lsp config, that we'll highlight more unused vars, hopefully will remove them then.